### PR TITLE
Update search functionality

### PR DESCRIPTION
**Pull request by blackeagle0188**

### CODE REVIEW

**Comment by trkaplan** [comment]
Typing an id and pressing enter / clicking search icon does not display the related plot.
Can you check it?
this is what I expect: from #102 
>  When I type a number of an NFT and press enter or click to the search icon It should show the one with that number.

**Comment by blackeagle0188** [comment]
> Typing an id and pressing enter / clicking search icon does not display the related plot. Can you check it? this is what I expect: from #102
> 
> > When I type a number of an NFT and press enter or click to the search icon It should show the one with that number.

Yeah, It displayed automatically NFT when input the number of NFT.
I have fixed it and you can check it.

**Comment by trkaplan** [comment]
I checked your deploy https://app-cf3035u16-citydao.vercel.app/ as well as this repo's vercel deploy (https://parcel-viewer-500hmx7nz-city-dao.vercel.app/)
I type 2322 in the box and click to search icon or enter key on my keyboard but I can't see the **plot popup** on map or **plot details** on the sidebar. 
Screen Shot 2022-06-28 at 20 52 08

File: packages/react-app/src/components/SidePanel.tsx [hunk 1]
```diff
@@ -46,36 +46,30 @@ export default function SidePanel({ contracts, injectedProvider }: Props) {
     }, [idFilter])
 
     return (
-        <div className="plot-tabs overflow-y-scroll">
+        <div className="plot-tabs overflow-auto">
```
Comment (trkaplan) [inline packages/react-app/src/components/SidePanel.tsx:49]: I hope this won't cause an unwanted horizontal scrollbar

**Review by trkaplan — APPROVED** [review]

**Pull request merged**